packages/flutter_sprites/lib/src/action.dart

```diff
@@ -34,19 +39,23 @@ abstract class Action {
     _finished = false;
   }
 
+  /// The total time it will take to complete the action, in seconds.
+  ///
+  ///     double myTime = myAction.duration;
```

Comment:
This just seems to be an example of assignment... I wouldn't bother including that here. If they've gotten to this point, they know how to do that. :-)